openshift/templates/pims-db/mssql-build.yaml

```diff
@@ -0,0 +1,133 @@
+# This is a build template for .NET Core 3.1
+# It will build a new image from the specified source control repo
+# that will run your application.
+kind: Template
```

Comment:
We now will use a custom built image for our database so that we can initialize it.